features/iotserver-feature/org.wso2.carbon.device.mgt.iot.feature/src/main/resources/jaggeryapps/store/extensions/app/store-device-type/app.js

```diff
@@ -68,9 +68,10 @@ app.server = function (ctx) {
                 secured: true
             }, {
                 url: 'devices',
-                path: 'device-api.jag',
-                secured: true
-            }, {
+                path: 'device-api.jag'
+                //TODO: uncomment this once this resolved https://wso2.org/jira/browse/STORE-1112
```

Comment:
No. Still we can't secure this jaggery api using ES framework due to STORE-1112. However we have dropped ES extended approach.